autoscheduler/frontend/src/components/CourseSelectColumn/CourseSelectColumn.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+import { RouteComponentProps } from '@reach/router';
+import { useSelector, useDispatch } from 'react-redux';
+import { Button } from '@material-ui/core';
+import * as styles from './CourseSelectColumn.css';
+import { RootState } from '../../redux/reducers';
+import { CourseCardArray, CustomizationLevel } from '../../types/CourseCardOptions';
+import CourseSelectCard from './CourseSelectCard/CourseSelectCard';
+import { addCourseCard } from '../../redux/actions';
+
+/**
+ * Renders a column of CourseSelectCards, as well as a button to add course cards
+ */
+const CourseSelectColumn: React.FC<RouteComponentProps> = () => {
+  const courseCards = useSelector<RootState, CourseCardArray>(
+    (state) => state.courseCards,
+  );
+
+  const dispatch = useDispatch();
+
+  const rows: JSX.Element[] = [];
+
+  // Add all of the course cards to rows to be displayed
+  for (let i = 0; i < courseCards.numCardsCreated; i++) {
+    if (courseCards[i]) {
+      rows.push(
+        <div
+          className={styles.row}
+          key={`courseSelectCardRow-${i}`}
+        >
+          <CourseSelectCard
+            key={`courseSelectCard-${i}`}
+            id={i}
+          />
+        </div>,
+      );
+    }
+  }
```

Comment:
![image](https://user-images.githubusercontent.com/28655462/77695170-53731b80-6f79-11ea-9336-dd1ef9f0205b.png)
Could you remove the bottom padding from the last course card so it touches the bottom of the div it's in?

Reply:
Like you want it so that it touches the Add Course button?

Reply:
I think remove the padding from the last course card using a CSS pseudo-selector (last-of-kind or something) and then add marginBottom to the entire container div

Reply:
Yeah I meant the way Adel said, so that we still have the spacing between the end of the course card component and the add course button, and the scroll bar doesn't touch the button either